www/rustup.css

```diff
@@ -31,49 +31,69 @@
     src: local('Work Sans Medium'), url("fonts/WorkSans-Medium.ttf") format('truetype');
 }
 
+@font-face {
```

Comment:
I think the font changes should constitute a separate commit.

Reply:
Done.